docs/en/observability/profiling-upgrade.asciidoc

```diff
@@ -19,8 +20,9 @@ To upgrade Universal Profiling, you need to:
 
 . Perform a stack upgrade in {ecloud}
 . Stop profiling data ingestion
-. Delete existing profiling data
-. Setup Universal Profiling from scratch
+. *When upgrading from version 8.8.0 or more recent*: Migrate existing profiling data
+. *When upgrading from versions prior to 8.8.0*: Delete existing profiling data and setup Universal Profiling from scratch
+

```

Comment:
I know it's not part of this PR, but I spotted a potential problem a little further down.
When we say `Perform a stack upgrade`, I think we should include a step that includes `Stop profiling data ingestion` between point 3 and 4.

Reply:
I can swap the steps, so it says:


1. Stop profiling data ingestion
2. Perform a stack upgrade in Elastic Cloud
3. When upgrading from version 8.8.0 or more recent: Migrate existing profiling data
4. When upgrading from versions prior to 8.8.0: Delete existing profiling data and setup Universal Profiling from scratch
5. Start profiling data ingestion

I'll also reorder the sections accordingly.